docs/guides/burst_insar_product_guide.md

```diff
@@ -0,0 +1,195 @@
+{% extends "guides/insar_product_guide_template.md" %}
+
+{% block header %}
+# Sentinel-1 Burst InSAR Product Guide
+
+This document is a guide for users of Sentinel-1 Burst Interferometric Synthetic Aperture Radar (InSAR) products generated by the Alaska Satellite Facility (ASF). 
+
+### Sentinel-1 Bursts
+
+Data from the Sentinel-1 mission that is suitable for use in interferometry has historically been packaged into Single Look Complex (SLC) products. These SLCs include three subswaths, each containing many individual bursts. The framing of the SLCs is not consistent through time, so when using SLCs as the basis for InSAR, scene pairs do not always fully overlap. 
```

Comment:
```suggestion
Single Look Complex (SLC) data from the Sentinel-1 mission that is suitable for use in interferometry has historically been packaged into Interferometric Wide (IW) SLC products. These IW SLC products include three subswaths, each containing many individual burst SLCs. The framing of the IW SLCs is not consistent through time, so when using IW SLCs as the basis for InSAR, scene pairs do not always fully overlap. 
```

SLC is a shorthand for the Sentinel-1 product format, but it is also the type of data contained within these products. I think it's an important to make the distinction between the products themselves and the data they contain so that it's clear that the burst products are also SLCs, just of a different size.

Reply:
Do we still limit InSAR processing to just IW? Or can users submit EW SLCs for processing? Just making sure we don't need to broaden our references to IWs to *also* include EWs.

Reply:
We don't support EW. The HyP3 API requires "IW" be in the burst granule names. The hyp3-isce2 plugin failed to process the first two EW bursts I tried, one complaining "swath must be 1,2,3" and another complaining that no annotation .xml could be found in our mocked .SAFE directory.

Reply:
Yep of the time being we only support IW.